scss/forms/_form-control.scss

```diff
@@ -30,6 +30,12 @@
     }
   }
 
+  &[type="search"] {
+    &::-webkit-search-cancel-button {
+      cursor: pointer;
+    }
+  }
```

Comment:
Hey there,

Thanks for the contribution.
Is there any way to group this block with the following line in `_reboot.scss`? It would make more sense to me down there. 
https://github.com/twbs/bootstrap/blob/88161777a702a05a5810a2bcf25590ab75cb4036/scss/_reboot.scss#L534

Reply:
it seems we're overall inconsistent already here (just see the preceding few lines in the CSS for file inputs). so i'd say at this stage, might as well leave it here?